### PR TITLE
Migrate rcs-webpack-plugin to Webpack 5.

### PR DESCRIPTION
**Pull request by chibibirdie**
This PR covers migrating to the latest Webpack version. I'm not sure you desire backwards compatibility (WP4), but I can look into it if so. Regardless it would be great to have support for the latest version of Webpack!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rcs-webpack-plugin",
-  "version": "1.4.2",
+  "version": "1.5.0",
```
Comment (JPeer264) [inline package.json]: Please do not upgrade the version

**Comment by JPeer264** [comment]
Thanks for upgrading. A backwards compatibility is not necessary in this case. I will upgrade the major version.

File: package.json (hunk 1 above)
Comment (chibibirdie) [reply package.json]: Reverted the change [here](https://github.com/JPeer264/rcs-webpack-plugin/pull/23/commits/0ba148e23e96f9bcb0b65f03e80203b6df1de0a0)

**Comment by chibibirdie** [comment]
@JPeer264 I've fixed the tests. They should all be passing now whenever you have a chance to re-run the Node CI pipeline.

**Comment by JPeer264** [comment]
Thanks a lot for contributing.

**Comment by JPeer264** [comment]
@chibibirdie you can use v`2.0.0` now with Webpack 5.

**Pull request merged**